demos/evp/aesccm.c

```diff
@@ -47,79 +48,157 @@ static const unsigned char ccm_tag[] = {
     0x14, 0xdb, 0x50, 0x6d
 };
 
-void aes_ccm_encrypt(void)
+int aes_ccm_encrypt(void)
 {
+    int ret = 0;
     EVP_CIPHER_CTX *ctx;
+    EVP_CIPHER *cipher;
```

Comment:
`EVP_CIPHER *cipher = NULL;`
Else your first `goto err;` will crash...

Reply:
I am not seeing these changes that you are marked as resolved?

Reply:
Just preparing a commit now